### PR TITLE
#1685 New Script - ADMIN - Individual On Demand Notices

### PR DESCRIPTION
**Pull request by C-Love**
This is the new script creation. We will have another pull request with updates to CLOS, and to retire the 3 scripts this new script replaces.

### CODE REVIEW

File: admin/individual-on-demand-notices.vbs [hunk 1]
```diff
@@ -0,0 +1,531 @@
...
+
+If memo_to_send = "APPL - Appt Notice" Then
+	last_contact_day = dateadd("d", 30, application_date)
+	If DateDiff("d", interview_date, last_contact_day) < 1 then last_contact_day = interview_date
+
+	end_msg = "Application - Appointment Notice" & vbCr & "Application Date: " & application_date & vbCr & "Interview Due: " & interview_date & vbCr & "Last Contact Day :" & last_contact_day
+	Call create_appointment_letter_notice_application(application_date, interview_date, last_contact_day)
+
+	Call confirm_memo_created_today(notc_confirm)
+	MsgBox "notc_confirm - " & notc_confirm
```
Comment (IlseFerris) [inline admin/individual-on-demand-notices.vbs:452]: Line 452 - msgbox needs to be removed.

File: admin/individual-on-demand-notices.vbs [hunk 2]
```diff
@@ -0,0 +1,531 @@
...
+		err_msg = ""
+		Dialog Dialog1
+		cancel_without_confirmation
+
+		If memo_to_send = "RECERT - APPT Notice" Then
+			If IsDate(interview_end_date) = False Then err_msg = err_msg & vbCr & "* Enter the date the interview is due by for the Notice to the resident."
+			If IsDate(last_day_of_recert) = False Then err_msg = err_msg & vbCr & "* Enter the last day of the certification period."
+		End If
+		If memo_to_send = "RECERT - NOMI" Then
+			If IsDate(interview_end_date) = False Then err_msg = err_msg & vbCr & "* Enter the date the interview is due by for the Notice to the resident."
```
Comment (IlseFerris) [inline admin/individual-on-demand-notices.vbs:350]: Line 350: Wrong variable here, not allowing me to pass.

**Review by IlseFerris — CHANGES_REQUESTED** [review]
See comments below. I was able to test all options, and all passed save for RECERT - NOMI.

**Comment by C-Love** [comment]
Updated with changes requested.

**Review by IlseFerris — APPROVED** [review]
I fudged the nomi date to test this and it worked as intended. I approve thee!

**Pull request merged**